db/schema.rb

```diff
@@ -23,6 +23,33 @@
     t.index ["trackable_id", "trackable_type"], name: "index_activities_on_trackable_id_and_trackable_type"
   end
 
+  create_table "boards", force: :cascade do |t|
```

Comment:
The schema.rb has some changes a part from including the new tables.
Some columns are reverted from `bigint` to `integer`.
But this happens if we generate the scheme with `database.yml` pointing to sqlite.
And since in CE we are using sqlite, seems like the correct thing to do ?
CI seems ok with this change.